services/CodeforcesService.py

```
@@ -0,0 +1,148 @@
+                handles.append(party_member["handle"])
+
+            for handle in handles:
+                if not (registered_users_with_handle.get(handle) is not None):
+                    continue
+
+                db_record = self.database_worker.find_one("proceeded-results", {
+                    "platform": "codeforces", 
+                    "contestId": contest['id'], 
+                    "proceeded_handle": handle 
+                })
+                if not (db_record is None):
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)